website/docs/Installation.md

```diff
@@ -1,10 +1,51 @@
 # Installation
 
-## Setup Virtual Environment
+## Option 1: Install and Run AutoGen in Docker
+#### Step 1. Install [docker](https://www.docker.com/).
+
+Docker is recommended as a properly built docker image could provide isolated and consistent environment to run your code securely across platforms. Install docker following [this instruction](https://docs.docker.com/get-docker/).
+
+#### Step 2. Build a docker image
+
+AutoGen provides [dockerfiles](https://github.com/microsoft/autogen/tree/main/samples/dockers/) that could be used to built docker images. Use the following command line to build a docker image named `autogen_img` (or some other name you like) one of the provided dockerfiles named `Dockerfile.baes`:
+```
+docker build -f sample/docker/Dockerfile.base -t autogen_img https://github.com/microsoft/autogen.git
```

Comment:
Note that this command is running when inside the root directory.

Reply:
Not necessarily? It will pull the dockerfile from the autgen github following the specified file path.

Reply:
Oh right! I missed the github link.